producers-commissioning/src/main/java/uk/gov/moj/sdt/producers/comx/config/bulkfeedback/BulkFeedbackConfigCOne.java

```diff
@@ -0,0 +1,902 @@
+                ""
+            ),
+            Lists.newArrayList(
+                "USER_REQUEST_ID_c69",
+                "mcolClaimStatusUpdate",
+                "Rejected",
+                "69",
+                "Defendant cannot be specified for an update type of Settled or Discontinued.",
+                ""
+            ),
+            Lists.newArrayList(
+                "USER_REQUEST_ID_c70",
+                "mcolClaimStatusUpdate",
+                "Rejected",
+                "71",
+                "Invalid status update requested for this defendant.",
+                ""
+            ),
+            Lists.newArrayList(
+                "USER_REQUEST_ID_c71",
+                "mcolClaimStatusUpdate",
+                "Rejected",
+                "71",
+                "Paid in full date specified in error.",
+                ""
+            ),
+            Lists.newArrayList(
+                "USER_REQUEST_ID_c72",
+                "mcolClaimStatusUpdate",
+                "Rejected",
+                "72",
+                "Paid in full date missing.",
+                ""
+            ),
+            Lists.newArrayList(
+                "USER_REQUEST_ID_c73",
+                "mcolClaimStatusUpdate",
+                "Rejected",
+                "73",
+                "Paid in full date cannot be in the future.",
+                ""
+            ),
+            Lists.newArrayList(
+                "USER_REQUEST_ID_c75a",
+                "mcolClaim",
+                "Rejected",
+                "75",
+                "Rejected by CCBC - rejection number 24.",
+                ""
+            ),
+            Lists.newArrayList(
+                "USER_REQUEST_ID_c75b",
+                "mcolJudgment",
+                "Rejected",
+                "75",
+                "Judgement: Rejected by CCBC - rejection number 6.",
+                ""
+            ),
+            Lists.newArrayList(
+                "USER_REQUEST_ID_c75c",
+                "mcolWarrant",
+                "Rejected",
+                "75",
+                "Warrant: Rejected by CCBC - rejection number 6.",
+                ""
+            ),
+            Lists.newArrayList(
+                "USER_REQUEST_ID_c75d",
+                "mcolJudgmentWarrant",
+                "Rejected",
+                "75",
+                "Judgment: Rejected by CCBC - rejection number 6.",
+                """
+                     <![CDATA[
+                                                <ind:mcolResponseDetail xmlns:ind="http://ws.sdt.moj.gov.uk/2013/sdt/BulkFeedbackResponseSchema" xmlns:mresp="http://ws.sdt.moj.gov.uk/2013/mcol/ResponseDetailSchema">
+                                                    <mresp:judgmentWarrantStatus>Judgment rejected by CCBC</mresp:judgmentWarrantStatus>
+                                                </ind:mcolResponseDetail>
+                                            ]]>
```

Comment:
Reduce indentation of text in text block (see previous examples).

Reply:
done